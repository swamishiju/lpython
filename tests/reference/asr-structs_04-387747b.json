{
    "basename": "asr-structs_04-387747b",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/structs_04.py",
    "infile_hash": "1e20c2ac044ab88183c50ecb481ac7c50992ed622f8bb94772c6df25",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-structs_04-387747b.stderr",
    "stderr_hash": "c1c7964cb8fa08afb544b22721a1a600cfaec090af51af4b5220d4b5",
    "returncode": 1
}