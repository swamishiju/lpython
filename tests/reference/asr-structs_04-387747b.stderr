Traceback (most recent call last):
[2m  File "[0m[1m[35m$DIR/lpython.cpp[39m[0m[2m", line 2135[0m
[2m  File "[0m[1m[35m$DIR/lpython.cpp[39m[0m[2m", line 216[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 9165[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 9097[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 5195[0m
[2m  File "[0m[1m[35m$DIR/python_ast.h[39m[0m[2m", line 1909[0m
[2m  File "[0m[1m[35m$DIR/python_ast.h[39m[0m[2m", line 1776[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 5385[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 5361[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 5149[0m
[2m  File "[0m[1m[35m$DIR/python_ast.h[39m[0m[2m", line 1909[0m
[2m  File "[0m[1m[35m$DIR/python_ast.h[39m[0m[2m", line 1782[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 5457[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 2896[0m
[2m  File "[0m[1m[35m$DIR/python_ast.h[39m[0m[2m", line 1936[0m
[2m  File "[0m[1m[35m$DIR/python_ast.h[39m[0m[2m", line 1825[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 9080[0m
[2m  File "[0m[1m[35m$DIR/semantics/python_ast_to_asr.cpp[39m[0m[2m", line 1335[0m
[2m  File "[0m[1m[35m$DIR/libasr/src/libasr/asr_utils.h[39m[0m[2m", line 4245[0m
    return types_equal(x, y, x_expr, y_expr, check_for_dimensions);
[2m  File "[0m[1m[35m$DIR/libasr/src/libasr/asr_utils.h[39m[0m[2m", line 3959[0m
    ASRUtils::get_struct_sym_from_struct_expr(a_expr)));
[2m  File "[0m[1m[35m$DIR/libasr/src/libasr/asr_utils.h[39m[0m[2m", line 3959[0m
    ASRUtils::get_struct_sym_from_struct_expr(a_expr)));
[2m  Binary file "[0m[1m[35m/usr/lib/system/libsystem_platform.dylib[39m[0m[2m", local address: 0x1804ac623[0m
Segfault: Signal SIGSEGV (segmentation fault) received
